crates/state/src/chain_state.rs

```diff
@@ -30,37 +30,16 @@ pub struct Chainstate {
     /// checkpoint 0, moving us into checkpoint period 1.
     pub(crate) epoch: u64,
 
-    /// Rollup's view of L1 state.
-    pub(crate) l1_state: l1::L1ViewState,
+    /// Epoch-level state that we only change as of the last block of an epoch.
+    // TODO this might be reworked to be managed separately
+    pub(crate) epoch_state: EpochState,
 
     /// Pending withdrawals that have been initiated but haven't been sent out.
     pub(crate) pending_withdraws: StateQueue<bridge_ops::WithdrawalIntent>,
 
     /// Execution environment state.  This is just for the single EE we support
     /// right now.
     pub(crate) exec_env_state: exec_env::ExecEnvState,
```

Comment:
Perhaps, could you expand on what the value of this would be?  I'm trying to be as data-oriented as possible here since I anticipate some refactors in the future here when we support async EE consensus.